### ISSUE TITLE
Captions doesn't handle national characters

### ISSUE THREAD

**Issue by setnicka** [derf/feh#163]
When in caption entry mode and trying to write some caption: All english and special characters (,.?!"@#$%^&*) works, but when trying to entry some national character (for example in Czech - letter like á, é, í, ě, š, č, ...) nothing happens.

Tested with the Czech and German keyboard, no national characters works.

**Comment by derf** [comment]
True. Unfortunately, feh has virtually no wide character support, so fixing this is lots of work. It should happen eventuelly, though.

**Comment by derf** [comment]
Resolved in 278802fcf41f3d79140f06720fdda98e938f2d65 – though feh's default font only has a limited set of supported characters, so that one will need work as well. Leaving this issue open for now.
